spec/unit/shoulda/matchers/active_record/define_enum_for_matcher_spec.rb

```diff
@@ -862,7 +939,17 @@ def build_record_with_enum_attribute(
       alias_attribute attribute_alias, attribute_name
     end
 
-    model.enum(enum_name => values, _prefix: prefix, _suffix: suffix)
+    params = {
+      enum_name => values,
+      _prefix: prefix,
+      _suffix: suffix,
+    }
+
+    if rails_version =~ '~> 6.0'
```

Comment:
Hi @dewyze, I'm wondering if this should be `rails_version >= '6.0'` instead? I would imagine Rails 7 has the same change as Rails 6 did.

Reply:
@mcmire annoyingly, in rails 7, they removed the leading underscore. https://api.rubyonrails.org/classes/ActiveRecord/Enum.html

So this won't work for rails 7, but there are no appraisals yet, so i wasn't going to worry about that.

Reply:
Ah, no problem then!